clients/include/testing_axpy.hpp

```diff
@@ -102,21 +112,20 @@ void testing_axpy_bad_arg()
 template<typename T>
 rocblas_status testing_axpy(Arguments argus)
 {
-
     rocblas_int N = argus.N;
     rocblas_int incx = argus.incx;
     rocblas_int incy = argus.incy;
     T alpha = argus.alpha;
 
-    T *dx, *dy;
+    T *dx = NULL, *dy = NULL, *dy_devptr = NULL, *alpha_devptr = NULL;
 
     rocblas_handle handle;
     rocblas_status status;
     status = rocblas_create_handle(&handle);
     verify_rocblas_status_success(status,"ERROR: rocblas_create_handle");
 
     if(status != rocblas_status_success) {
-        rocblas_destroy_handle(handle);
+        CLEANUP();
```

Comment:
See reply above, this will require a change throughout the test code, and it should likely be in a separate pull request.